src/css/dark.css

```diff
@@ -747,6 +736,4 @@ span.subcomponent {
 
 .a-canvas.state-dragging {
   cursor: grabbing;
-  cursor: -moz-grabbing;
```

Comment:
impressive, Firefox stopped prefixing this in Firefox 27.0. Chrome still vendor-prefixes this.